source/installation/tango-on-linux.rst

```diff
@@ -135,6 +135,19 @@ or if you need specifically Python2
        $> apt-get python-pytango
 
 
+CentOS
+------
+RPM packages are available for RedHat based systems in `the MAX-IV's repository <http://pubrepo.maxiv.lu.se/rpm/el7/x86_64/>`_.
+Use yum to install them e.g. to install the TANGO database and test device server:
+
+.. code-block:: console
+
+   $> sudo yum install mariadb\
+      sudo apt install libtango9 tango-db tango-test
```

Comment:
It seems, it should be `yum` instead of `apt` ;-)

Reply:
Thanks for review, fixed.